saleor/graphql/checkout/types.py

```diff
@@ -233,6 +236,22 @@ class Meta:
         interfaces = [graphene.relay.Node, ObjectWithMetadata]
         filter_fields = ["token"]
 
+    @staticmethod
+    def resolve_shipping_address(root: models.Checkout, info):
+        return (
+            AddressByIdLoader(info.context).load(root.shipping_address_id)
+            if root.shipping_address_id
+            else None
+        )
+
+    @staticmethod
+    def resolve_billing_address(root: models.Checkout, info):
+        return (
+            AddressByIdLoader(info.context).load(root.billing_address_id)
+            if root.billing_address_id
+            else None
+        )
```

Comment:
Same here.

Reply:
I removed it because this doesn't affect on performance.